examples/tutorials/04_linear_models_example.ipynb

```diff
@@ -24,17 +24,21 @@
     "import warnings\n",
```

Comment:
<p>Define a pipeline config under the same name in two variants (torch- and tf-based) thus allowing a user to switch between frameworks by commenting out a cell quickly rather then retyping a pipeline config variable name in several places</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>